siemens/siemens_common.mk

```diff
@@ -201,13 +201,27 @@ batch: $(PARAMETER_DONE) $(PRESIM_GOAL) $(presim_hook)
 	     echo -e "$(GREEN)# Simulation successful $C"; \
 	 fi;
 
+## Run autocheck in console only
+.PHONY: autocheck_batch
+autocheck_batch: $(AC_DONE)
+	@-$(HDL_BUILD_PATH)/siemens/ac_pretty.sh '$(AC_REPORT)';
```

Comment:
The `-` here causes `make` to ignore the error if the return code is unsuccessful. That seems wrong -- is there a reason for it?

Reply:
if Make exits then it won't run the gui.  The error is for a potential nightly or jenkins thing but I don't want it to stop the gui from launching.